tests/Aspire.Hosting.Tests/Utils/LoggerNotificationExtensions.cs

```diff
@@ -77,21 +74,23 @@ await foreach (var resourceEvent in resourceNotificationService.WatchAsync(cance
                 if (loggingResourceIds.Add(resourceId))
                 {
                     // Start watching the logs for this resource ID
-                    logWatchTasks.Add(WatchResourceLogs(tcs, resourceId, logTexts, resourceLoggerService, cancellationToken));
+                    logWatchTasks.Add(WatchResourceLogs(tcs, resourceId, logTexts, resourceLoggerService, cancellationTokenSource));
                 }
             }
-
-            await Task.WhenAny(logWatchTasks).ConfigureAwait(false);
+        }
+        catch (OperationCanceledException)
```

Comment:
Is this restricted to the case where `cancellationTokenSource.IsCancellationRequested==true`?